class AlreadyExists(Exception):
    pass


class DoesNotExist(Exception):
    pass
